docs/reference/components/sinks.cue

```diff
@@ -410,7 +410,7 @@ components: sinks: [Name=string]: {
 						buffers_batches: {
 							title: "Buffers & batches"
 							body: #"""
-									![Buffers and Batches](/optimized_svg/buffers-and-batches-serial_538_160.svg)
+									{{< svg "buffers-and-batches-serial.svg" }}
```

Comment:
Will this continue to render properly with the current site? I just fixed an issue in #7010 because it wasn't working.

Reply:
No, all of these changes are for the brand new site and should be seen as on completely different track from that PR submitted, which you should feel free to merge

Reply:
Actually, I'm going to close this PR, as I'm going to move the website build into this repo, which makes this PR unnecessary